### ISSUE TITLE
Display bike lanes

### ISSUE THREAD

**Issue by aronkvh** [organicmaps/organicmaps#4194]
OM displays cycleways by default, so the user assumes it shows bike infrastructure - however it doesn't show bike lanes on regular roads.
Displaying `cycleway=lane` tags would make it more consistent. While browsing the map, seeing which streets have a bike lane is useful information even when not planning a route yet.
These are what I mean: 
![kép](https://user-images.githubusercontent.com/57704946/210618943-066b2d39-db97-495c-991a-9bf2ccf4f7d1.png)

[cycleway](https://wiki.openstreetmap.org/wiki/Key:cycleway?uselang=en-GB) 	[lane](https://wiki.openstreetmap.org/wiki/Tag:cycleway=lane?uselang=en-GB)

**Comment by biodranik** [comment]
Wouldn't it pollute the "general" map style?

It is definitely a good idea for the specialized bike map style.

**Comment by aronkvh** [comment]
> Wouldn't it pollute the "general" map style?
> 
> It is definitely a good idea for the specialized bike map style.

IMO it's useful enough to be worth adding to the base style as well in a non-distracting way. Some bike paths being shown and some not is confusing, but removing all of them from the base view doesn't make sense either so....

**Comment by biodranik** [comment]
Can you please provide several OM screenshots and coordinates to test it? Displaying bike lanes unobtrusively and consistently may be a good idea.

**Comment by Sergey82K** [comment]
> and coordinates to test it?

For example, [marked bike lanes at road sides](url="https://www.openstreetmap.org/#map=19/50.44973/30.43382") (cycleway=lane). Near roads crossing, the bike lanes combined with car lanes (cycleway=shared_lane).

https://www.openstreetmap.org/#map=19/50.44973/30.43382

**Comment by aronkvh** [comment]
sure
"Üdvözlöm!

Jelenleg itt vagyok: Rákóczi Ferenc utca Kattintson erre a hivatkozásra
om://44Fj5I7OXW/Rákóczi_Ferenc_utca vagy erre
https://omaps.app/44Fj5I7OXW/Rákóczi_Ferenc_utca, hogy lássa a helyet a
térképen."
^This text is unnecessary when sharing btw.

The screenshot is from the same street I included in the description.

Alexander Borsuk ***@***.***> ezt írta (időpont: 2023. jan.
6., P 8:19):

**Comment by aronkvh** [comment]
![Uploading Screenshot_2023-01-26-18-02-34-03_10a7d9276315e62e56e7300dfec6be06.jpg…]()
Another example. Here it looks like the cycleway ends, even tough it continues in a straight line.

**Comment by aronkvh** [comment]
![Uploading Screenshot_2023-01-26-18-02-34-03_10a7d9276315e62e56e7300dfec6be06.jpg…]()
Another example. Here it looks like the cycleway ends, even tough it continues in a straight line.
47.095015, 17.912589

**Comment by biodranik** [comment]
Looks like the screenshots were not uploaded successfully...

**Comment by aronkvh** [comment]
https://www.openstreetmap.org/way/97808048
I'll try attaching it again. 
![kép](https://user-images.githubusercontent.com/57704946/214917812-e79044af-7c68-488a-8052-0452e342ff9b.png)

**Comment by g-normand** [comment]
Hello,
I have the same problem 
If it's possible to add cycleways to Organic Naps, that would be awesome.

Here is the case : https://www.openstreetmap.org/way/523917171#map=17/-2.90447/-79.01100

Here is the cycleway on OSM : 
![IMG_20230331_232638_967_edit_150061659718246](https://user-images.githubusercontent.com/17318506/229265616-d2da8203-c7b5-4096-bb2b-1b2522f0dcba.jpg)

But nothing shown on Organic Maps
![IMG_20230331_232641_959_edit_150073369488556](https://user-images.githubusercontent.com/17318506/229265610-1fd9d96d-9388-4999-9967-f336a2b0bcc7.jpg)

Thanks
Guillaume

**Comment by raar1** [comment]
OM currently renders only fully physically separated cycleways (that are separately mapped with e.g. `highway=cycleway`). These appear as solid pink lines. This is extremely useful, especially if you have children or are otherwise not into "vehicular cycling", because you can at a glance see where the "real" cycle paths are.

I am not against rendering unprotected painted lanes etc (which `cycleway=lane` represents) but this is a very significant difference in level of infrastructure and safety, so should not be rendered the same way. Perhaps a dashed pink line instead of a solid pink line could work?

As a second point, note that if OM is to render `cycleway=lane` then it should definitely render `cycleway=track` too (and IMO track the latter is much more important).

From <https://wiki.openstreetmap.org/wiki/Tag:cycleway%3Dtrack>:

> A cycle track is separated from the road by curbs, parking lots, grass verges, trees or another physical barrier, but is running parallel to and next to the road.

This makes `cycleway=track` equivalent to the pink cycleways that OM currently renders, as they are protected and separated from the road. In countries like Belgium and Denmark it appears to be very common not to map cycle paths separately, and `cycleway=track` is the standard way of mapping physically separated cycle paths that run parallel to the road.

In summary, I would propose to render `cycleway=track` as solid pink lines and `cycleway=lane` as e.g. a dotted line. Note that OpenCycleMap essentially does this already:

![image](https://github.com/organicmaps/organicmaps/assets/2650951/47717800-a992-40af-ad94-73585d6987a3)

<https://www.opencyclemap.org/?zoom=18&lat=51.14959&lon=5.59579&layers=B0000>

The above location shows an example of both tags (cycle infra is in blue)

**Comment by biodranik** [comment]
@pastk is it hard to add cycleway=track?

**Comment by pastk** [comment]
> @pastk is it hard to add cycleway=track?

Depends on how we'd want to render it. If render on the sides of the road - we don't have this capability yet.
If render as a centerline above the road (i.e. regardless if its on both sides of the road or one side only..) then its easier.

(it seems to be processed right by the routing engine)

**Comment by jcharaoui** [comment]
In Montreal, Canada, a lot of recent high quality, fully protected cycleway infrastructure has been mapped using `cycleway=track` and is thus missing from the OM cycling view, making it almost unuseable in that locality to appropriately plan bike trips without local knowledge.
